### DISCUSSION TITLE
Unable to authenticate

### DISCUSSION THREAD

**Discussion by chilldenpa** [community/community#50190] [GitHub Education]
I have been unable to pass, I have tried all the supporting documents, including the school certificate in English, campus card and student ID card, admission letter, student status certificate from the Ministry of Education

**Comment by neutrinomuon** [comment]
Hello! To assist you better, could you please provide more information about what you are looking for? Are you interested in obtaining the GitHub Student Developer Pack?

If so, I believe you have ensured that you are eligible and have followed **all** the guidelines from https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-global-campus-for-students/apply-to-github-global-campus-as-a-student ?

I will add them below again:

**REQUIREMENTS**

- Be currently enrolled in a degree or diploma granting course of study such as a high school, secondary school, college, university, -homeschool, or similar educational institution
- Have a verifiable school-issued email address or upload documents that prove your current student status
- Have a [GitHub personal account](https://docs.github.com/en/get-started/signing-up-for-github/signing-up-for-a-new-github-account)
- Be at least 13 years old

I think the reason could be that your enrollment status is not clearly stated in your documents. For example, an admission letter is **not** enough evidence of your enrollment because you may have been transferred to a different school or completed your studies already. To avoid confusion, it's advisable to get a signed and verified letter of enrollment from your school coordinator that includes information such as your enrollment status, admission dates, and expected graduation date. In addition, it's important to make sure that you have provided a school email ID, as its absence could result in significant issues. As I mentioned earlier, I recommend obtaining a digitally signed and verified letter from your school coordinator with the corresponding info to prevent any potential problems.

I hope this helps.

**Reply by chilldenpa** [reply]
HI there:
     I have provide my  verifiable school-issued email address and a school certificate (with official seal) that is common in our place, but it seems that it cannot pass the verification of github, and it is rejected every time it is submitted. May I email you privately for these proof materials? Thank you so much for helping me

**Reply by mufeedcm** [reply]
same here uploaded the two id's I got for the schools but still 😞 **this is the 3rd-time rejection**

**Reply by neutrinomuon** [reply]
Hi... To ensure the security and confidentiality of information it's best to send such material directly to GitHub support rather than to any user, including me. If you require additional assistance, you can contact GitHub support by submitting a request through their contact form at the following link: https://support.github.com/contact

When it comes to your documents, it's important to note that a school certificate with an official seal seems not to be enough to demonstrate that you are currently enrolled in a school or educational institution. To establish your current enrollment status, you might need to acquire a more recent document like an enrollment verification letter from your school or an official transcript that showcases your current courses and enrollment status. Anyway, if you still need to try to reach support to confirm the specific requirements.

I hope now it is clearer.
